src/app/Http/Controllers/API/AuthController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Resources\UserResource;
+use App\User;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\Hash;
+
+/**
+ * Class AuthController
+ * @package App\Http\Controllers\API
+ * @group Authentication
+ * All routes that need authentication should contain a bearer token in the headers
+ * using the following synthax:<br>
+ * `Authorization: Bearer <token>`
+ */
+class AuthController extends Controller
+{
+    /**
+     * Log the user in
+     *
+     * @responseFile responses/login.get.json
+     *
+     * @bodyParam email email required email of the user to connect
+     * @bodyParam password string required the password related to the user
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function login(Request $request)
+    {
+        $http = new \GuzzleHttp\Client;
```

Comment:
Since you're using it twise, wouldn't it be better to have a `use \GuzzleHttp`? So like that you don't need to specify it in the code.

Reply:
it's only used once XD

Reply:
You're using Guzzle at line 46 --> used twice

Reply:
yes but it's not Guzzle\Client